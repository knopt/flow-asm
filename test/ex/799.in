2
24
-8.734878091941667
7.244513644174887 0.34487663991344064 
-2.6178232586665917 -2.4174639124677917 
-3.277557710940961 0.8345292575403427 
3.492924931788348 -9.54641529604585 
-9.620780857845695 -4.225704495956604 
3.457114618612053 -4.691454064626404 
-5.339608738700283 1.3208001724490277 
9.834057368601236 -3.104420386740583 
0.11533549306758495 2.6854405573907094 
3.6619757681794525 -1.9800014417107583 
7.584693867294327 -5.2884971630193895 
7.5556282477747985 -1.6297143659461515 
-5.375484087286511 -4.362214000138316 
-5.686415612073943 -1.7804484006907408 
-2.3854847828942694 -2.8230606692905758 
-5.621670853114669 0.598465911720643 
-9.626205921341578 -9.933559500535157 
6.240364775426336 -3.8477963596956677 
9.530678180771805 -4.314283201944685 
-3.7323715339819863 -6.535632840106218 
-9.77699410942196 -5.9987208280135 
9.821739855248634 -8.767192508152439 
-4.104428669610427 -9.37859137020424 
2.817398653637488 -0.3786699595705567 

-5.531823311103176 5.543584691382364 -3.2113072184032276 -1.4154241546886137 -0.09369442831605568 -5.814542370509395 0.537013800384404 -1.3389341833666464 2.077883076421818 -0.2305725460414898 -1.9388464168138437 -1.265806820887395 0.81457576884935 -9.85576423479958 7.732841626720077 -9.848624134230386 -0.871537955296315 8.145255975061158 8.813739972747936 9.259176057808219 2.5145910661465507 3.835654759749879 -6.0312113446636895 3.608967973115602 
