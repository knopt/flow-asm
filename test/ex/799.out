219.84797700377248 -9.91508106037851 
-123.65289203948758 -130.08520292521703 
-162.075401329037 162.77027290608996 
309.9420511534576 -314.7036406066413 
-426.5389831619483 -41.08611972959921 
332.03864576563785 -83.73416098197694 
-296.8465371504832 57.64601995988645 
473.8185300305973 -213.94801523474743 
-117.17426673941128 45.48857921053521 
97.13727236354094 -164.9637050790067 
270.86345022111385 -368.9657395378993 
339.1229437223174 -73.80338017158849 
-166.4434722014064 -134.4650417838775 
-174.81778427093673 100.68331535671955 
96.7746794659052 3.817633811952332 
-120.13018468421868 292.1888854202527 
-412.957606955136 -336.7136912149289 
139.0969313828501 -101.11919029748687 
174.85768691156733 -207.19631134286078 
-344.4192483476313 -167.1392304847814 
-626.3175395496272 -102.10992094190888 
557.2460497039128 -221.8891984802838 
-306.11015923218775 -413.6445091294983 
133.65598460701332 82.86132482170513 
